app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -23,216 +25,248 @@ class RoomProfilePage extends ConsumerWidget {
-              sliver: SliverToBoxAdapter(
-                child: convo.when(
-                  data: (data) => Center(
-                    child: Text(
-                      data.topic() ?? '',
-                      style: Theme.of(context).textTheme.bodySmall,
-                      softWrap: true,
-                      textAlign: TextAlign.center,
+                SettingsSection(
+                  tiles: [
+                    SettingsTile(
+                      onPressed: (ctx) async {
+                        await showAdaptiveDialog(
+                          barrierDismissible: true,
+                          context: ctx,
+                          builder: (ctx) => DefaultDialog(
+                            height: MediaQuery.of(context).size.height * 0.5,
+                            title: topMenu,
+                            description: convo.when(
+                              data: (data) => MemberList(convo: data),
+                              loading: () => const Text('loading...'),
+                              error: (e, s) => Text('Error: $e'),
+                            ),
+                          ),
+                        );
+                      },
+                      title: Text(
+                        'Members',
+                        style: tileTextTheme,
+                      ),
+                      leading: const Icon(
+                        Atlas.accounts_group_people_thin,
+                        size: 18,
+                      ),
+                      trailing:
+                          const Icon(Icons.chevron_right_outlined, size: 18),
                     ),
-                  ),
-                  loading: () => const Text('loading...'),
-                  error: (e, s) => Text('Error: $e'),
+                  ],
                 ),
-              ),
-            ),
-            SliverPadding(
-              padding: const EdgeInsets.all(8.0),
-              sliver: SliverToBoxAdapter(
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  children: topMenu,
+                SettingsSection(
+                  title: Text(
+                    'Danger Zone',
+                    style: Theme.of(context).textTheme.titleSmall!.copyWith(
+                          color: Theme.of(context).colorScheme.badgeUrgent,
+                        ),
+                  ),
+                  tiles: [
+                    SettingsTile(
+                      title: Text(
+                        'Leave Room',
+                        style: tileTextTheme!.copyWith(
+                          color: Theme.of(context).colorScheme.badgeUrgent,
+                        ),
+                      ),
+                      leading: Icon(
+                        Atlas.trash_thin,
+                        size: 18,
+                        color: Theme.of(context).colorScheme.badgeUrgent,
+                      ),
```

Comment:
no `onPressed`? should probably have one that
1. confirms they want to leave
2. removes them from the room
3. routes somewhere else (`/chat`?)

if you don't want to do that before merge, then remove the section for now and put that into a ticket for later. But a button that again doesn't work isn't really acceptable to end users...

Reply:
I remember this wasn't already working though?? OR got removed somewhere in other PR's commit history. I'll add the functionality back in any case.